infrastructure/models.py

```diff
@@ -23,6 +23,22 @@ class Meta:
     def __str__(self):
         return self.name
 
+class OwnerStake(models.Model):
+    """ Percentage that owners own and also relation with PowerPlant and Organization """
+    power_plant = models.ForeignKey(
+        'PowerPlant',
+        models.CASCADE,
+        related_name = 'owner_stakes'
+    )
+    owner = models.ForeignKey(
+        'facts.Organization',
+        models.CASCADE,
+        related_name = 'owners_stakes'
+    )
+    percent_owned = models.FloatField(blank=True, null=True)
+
+    def __str__(self):
+        return self.owners
```

Comment:
This should be `return '{}'.format(self.owner)` or something like that. Otherwise, it throws an error that it's returning an object, rather than a string

Reply:
Or maybe `return '{} stake in {}'.format(self.owner, self.power_plant)`